app/src/main/java/ca/unb/mobiledev/reflexrevolution/detectors/TouchDetector.java

```diff
@@ -94,10 +95,14 @@ public boolean onTouch(View view, MotionEvent motionEvent) {
         DONT_TAP
     }
     public enum SwipeAction {
-        SWIPE_RIGHT,
-        SWIPE_LEFT,
-        SWIPE_UP,
-        SWIPE_DOWN
+        FLICK(Type.FLING),
+        SWIPE_RIGHT(Type.SWIPE),
+        SWIPE_LEFT(Type.SWIPE),
+        SWIPE_UP(Type.SWIPE),
+        SWIPE_DOWN(Type.SWIPE);
```

Comment:
We need to define the Type enum here as well. (Android studio should be giving you a bunch of errors because it doesn't know what `Type` is)
```java
public enum Type { FLING, SWIPE }
```